apps/web-client/src/lib/__testData__/debugData.sql

```diff
@@ -7,9 +7,9 @@ INSERT INTO book (isbn, title, authors, publisher, price) VALUES
 ('9780590353427', 'Harry Potter and the Sorcerer''s Stone', 'J.K. Rowling', 'Scholastic', 12.50);
 
 -- Suppliers
-INSERT INTO supplier (id, name, email, address) VALUES
-(1, 'BooksRUs', 'contact@booksrus.com', '123 Book St, New York, NY'),
-(2, 'NovelSupply Co.', 'support@novelsupply.co', '456 Fiction Ave, Los Angeles, CA');
+INSERT INTO supplier (id, name, email, address, customerId) VALUES
```

Comment:
Looks like the `.sql` file is not loaded anymore on the debug page, those changes should end up in [`debugData.ts`](https://github.com/librocco/librocco/blob/927d88f7ea32dd6bcfb12ebda04af08754e27b5e/apps/web-client/src/lib/__testData__/debugData.ts#L39)